third_party/silabs/efr32_sdk.gni

```diff
@@ -525,15 +525,10 @@ template("efr32_sdk") {
       defines += [ "SL_MATTER_PROVISION_FLASH=0" ]
     }
 
-    if (chip_enable_ble_rs911x) {
+    if (chip_enable_ble_siwx) {
       defines += [
-        "RSI_BLE_ENABLE=1",
-        "BLE_ENABLE=1",
-        "RSI_LITTLE_ENDIAN=1",
         "SLI_SI91X_ENABLE_BLE=1",
```

Comment:
if we are using the `args.gni`, this should be moved to the correct location in that case, `chip_enable_ble_siwx` is already present, lets use that.